packages/devtools_app/lib/src/screens/debugger/call_stack.dart

```diff
@@ -30,13 +29,13 @@ class _CallStackState extends State<CallStack> {
     final newController = Provider.of<DebuggerController>(context);
     if (newController == controller) return;
```

Comment:
this line is now wrong. Needs to be newController == _controller. Without that, this code should be crashing at runtime.

Reply:
Hah yes I discovered that when I tried to run it - fixed now, thanks!